frontend/src/models/Cluster.ts

```diff
@@ -0,0 +1,17 @@
+import { Criteria } from "./types/criteria.ts";
+import { Cluster } from "./types/cluster.ts";
+
+export default function createCluster(
+  title: string = "",
+  criteria: Criteria[] = [],
+  description: string = "Enter description",
+  id: number = crypto.getRandomValues(new Uint32Array(1))[0], // default to unique random number if not assigned by
```

Comment:
change to use uuid for react keys (check some of the other model files for reference!)

if we store clusters on the db, the backend will handle the id